tests/checkpointing/test_model_checkpoint.py

```diff
@@ -935,7 +927,8 @@ def assert_checkpoint_log_dir(idx):
         lightning_logs = tmpdir / "lightning_logs"
         actual = [d.basename for d in lightning_logs.listdir(sort=True)]
         assert actual == [f"version_{i}" for i in range(idx + 1)]
-        assert len(ckpt_dir.listdir()) == epochs
+        actual = [d.basename for d in ckpt_dir.listdir()]
+        assert len(actual) == epochs, actual
```

Comment:
```suggestion
        assert len(actual) == epochs
```

Reply:
This is on purpose, if the assertion fails, we don't know why

because it reports something like `2 != 1`, and you need to debug to see what's really in actual